src/lib/mongoBackend/MongoCommonRegister.cpp

```diff
@@ -273,7 +274,7 @@ static bool addTriggeredSubscriptions
       LM_T(LmtMongo, ("adding subscription: '%s'", sub.toString().c_str()));
 
       TriggeredSubscription* trigs = new TriggeredSubscription(
-        sub.hasField(CASUB_FORMAT) ? stringToFormat(getStringFieldF(sub, CASUB_FORMAT)) : JSON,
+        sub.hasField(CASUB_FORMAT)? stringToNotificationFormat(getStringFieldF(sub, CASUB_FORMAT)) : NGSI_V1_JSON,  // FIXME PR: hard to decide a default NotificationFormat ... JSON or NGSI_V2_NORMALIZED ?
```

Comment:
This is related with a previous comment. I'd sugges to use NGSI_V1_JSON plus a comment like:

// FIXME: this will be refactored once ctx availability notification formats get defined for NGSIv2"